scripts/genproto.sh

```diff
@@ -16,9 +16,14 @@ if ! [[ $(protoc --version) =~ "3.0.0" ]]; then
 fi
 
 export PATH=.:${PATH}
+cd $GOPATH/src/github.com/coreos/rkt
```

Comment:
A valid concern, and I've stumbled upon this myself. The problem is, that this script picks up the `GOPATH` created by our make framework (when invoked via `make protobuf`), which is `rkt/build-rkt-x.yy.z+git/gopath`.

This causes the go compiler to ignore the `rkt/vendor` directory, so I have to explicitly `cd rkt/build-rkt-x.yy.z+git/gopath/src/github.com/coreos/rkt`, in order to let Go pick up `rkt/build-rkt-x.yy.z+git/gopath/src/github.com/coreos/rkt/vendor` instead.

This line is not needed, when invoked outside of `make` (with your "normal" `GOPATH`). I also tried to `unset GOPATH` in the script, but Go doesn't like this either.